app/src/main/java/com/sdpteam/connectout/client_validation/EventCreationValidationUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.sdpteam.connectout.client_validation;
+
+import android.widget.EditText;
+import android.widget.RadioButton;
+
+public class EventCreationValidationUtils {
+
+    private static final int MIN_EVENT_TITLE_LENGTH = 5;
+    private static final int MIN_EVENT_DESCRIPTION_LENGTH = 20;
+
+    public static final String EVENT_TITLE_ERROR = "Title too short, should be at least " + MIN_EVENT_TITLE_LENGTH + " characters";
+    public static final String EVENT_DESCRIPTION_ERROR = "Description too short, should be at least " + MIN_EVENT_DESCRIPTION_LENGTH + " characters";
+
+    public static boolean isValidEventTitle(String title) {
+        return title.length() >= MIN_EVENT_TITLE_LENGTH;
+    }
+    public static boolean isValidEventDescription(String description) {
+        return description.length() >= MIN_EVENT_DESCRIPTION_LENGTH;
+    }
+
+    public static boolean eventCreationValidation(EditText eventTitleInput, EditText eventDescriptionInput) {
+        if(!isValidEventTitle(eventTitleInput.getText().toString())) {
+            eventTitleInput.setError(EVENT_TITLE_ERROR);
+            eventTitleInput.requestFocus();
+            return false;
+        }
+        if(!isValidEventDescription(eventDescriptionInput.getText().toString())) {
+            eventDescriptionInput.setError(EVENT_DESCRIPTION_ERROR);
+            eventDescriptionInput.requestFocus();
+            return false;
+        }
```

Comment:
I would argue to reduce here the code redundancy,
Typically using a private method `private static boolean handleValidationFailure(boolean condition, TextView view, String error); ` which does the following. this would enable the usage of a single boolean variable which checks for everything.

Typically having,
```
boolean isValid = handleValidationFailure(!isValidEventTitle(...),eventTitleInput, EVENT_TITLE_ERROR );
isValid &= handleValidationFailure(!isValidEventDescription(...), eventDescriptionInput, EVENT_DESCRIPTION_ERROR);
```

what do you think ?

Reply:
good point ! I have made a separate class which contains this method so I avoid copy pasting !